src/awsexplorer/activation.ts

```diff
@@ -92,25 +94,20 @@ async function registerAwsExplorerCommands(
                 outputChannel: lambdaOutputChannel,
                 onRefresh: () => awsExplorer.refresh(node.parent)
             }),
-        telemetryName: {
-            namespace: TelemetryNamespace.Lambda,
-            name: 'delete'
-        }
+        telemetryName: 'Command_aws.deleteLambda'
```

Comment:
Should this not be `lambda_delete`?

Reply:
Nope, there are no metric name translations. This is what the wrapper would output

Reply:
Oh I get what you mean now, fixed